includes/Commerce/Orders.php

```diff
@@ -82,10 +82,86 @@ public function create_local_order( Order $remote_order ) {
 	 * @param Order $remote_order Orders API order object
 	 * @param \WC_Order $local_order local order object
 	 * @return \WC_Order
+	 * @throws SV_WC_Plugin_Exception|\WC_Data_Exception
 	 */
 	public function update_local_order( Order $remote_order, \WC_Order $local_order ) {
 
-		// TODO: implement
+		// add/update items
+		foreach ( $remote_order->get_items() as $item ) {
+
+			$wc_product_id = $item['retailer_id'];
```

Comment:
The FB retailer ID isn't 1:1 with the Woo product ID. See `\WC_Facebookcommerce_Utils::get_fb_retailer_id()`.

Without a SKU, the retailer ID will be `wc_post_id_{product-id}`
With a SKU, it'll be `{SKU}_{product-id}`

Reply:
For products that are synced via API, we also have stored the concrete Facebook product ID to post meta, so that could be potentially used for quick retrieval as well, then fall back to matching the retailer ID.

Reply:
Updated in https://github.com/facebookincubator/facebook-for-woocommerce/pull/1515/commits/a721ed6f18b03e1364bf27b91c974d31609baf8a